Libraries/Components/Button.js

```diff
@@ -273,11 +278,14 @@ class Button extends React.Component<ButtonProps> {
         buttonStyles.push({backgroundColor: color});
       }
     }
-    const accessibilityState = {};
+    const accessibilityState =
+      disabled != null
```

Comment:
I'd suggest `const accessibilityState = disabled === true ? ... : ...` so it's `accessiblityState.disabled | disabled`.

Reply:
Sorry to follow up on this can we avoid creating a new object here?
```
const accessibilityState = disabled !== accessibilityState.disabled ? {...this.props.accessibilityState, disabled} : this.props.accessibilityState
```